dcat/index.html

```diff
@@ -2713,10 +2716,14 @@ <h3>Class: Dataset Series</h3>
             <tr><th>RDF Class:</th><td><a href="http://www.w3.org/ns/dcat#DatasetSeries"><code>dcat:DatasetSeries</code></a></td></tr>
             <tr><th class="prop">Definition:</th><td>A collection of datasets that are published separately, but share some common characteristics that groups them. <!--, and could also be made available as a single dataset.--></td></tr>
             <tr><th class="prop">Sub-class of:</th><td><a href="#Class:Dataset"><code>dcat:Dataset</code></a></td></tr>
+            <!--tr><th class="prop">Usage note:</th><td> Child datasets in dataset series SHOULD be represented as <code>dcat:Dataset</code>.<!--The use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.></td></tr-->
             <tr><th class="prop">Usage note:</th><td> Dataset series can be also soft-typed via property <code>dcterms:type</code> as in the approach used in [[?GeoDCAT-AP]], and adopted in [[?DCAT-AP-IT]] and [[?GeoDCAT-AP-IT]]).</td></tr>
             <tr><th class="prop">Usage note:</th><td>Common scenarios for dataset series include: time series composed of periodically released subsets; map-series composed of items of the same type or theme but with differing spatial footprints.</td></tr>
+            <tr><th class="prop">See also:</th><td><a href="#dataset-series-before-dcat3"></a></td></tr>
             </table>
 
+<p>For guidance on the use of this property, see <a href="#dataset-series-before-dcat3"></a>.</p>
```

Comment:
@andrea-perego : I wonder if we should point to the section #dataset-series instead of the subsection about  existing implementations.

```suggestion
<p>For guidance on the use of this property, see <a href="#dataset-series"></a>.</p>
```

Reply:
Agreed.